src/openvic-simulation/pop/Pop.cpp

```diff
@@ -206,7 +418,7 @@ bool PopManager::load_pop_type_file(
 		"issues", ZERO_OR_ONE, assign_variable_callback(issues_node),
 		"demote_migrant", ZERO_OR_ONE, expect_bool(assign_variable_callback(demote_migrant)),
 		"administrative_efficiency", ZERO_OR_ONE, expect_bool(assign_variable_callback(administrative_efficiency)),
-		"tax_eff", ZERO_OR_ONE, success_callback, // TODO - tax collection modifier
+		"tax_eff", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(tax_efficiency)),
 		"can_build", ZERO_OR_ONE, expect_bool(assign_variable_callback(can_build)),
 		"factory", ZERO_OR_ONE, expect_bool(assign_variable_callback(factory)),
 		"workplace_input", ZERO_OR_ONE, success_callback, // TODO - work out what these do
```

Comment:
meaningless

Reply:
What do you mean? None of these do anything?
```
workplace_input = -0.1
workplace_output = 0
starter_share = 1.0
```

Reply:
None of them have an effect

Reply:
starter share has to do with how much money pops that become capitalists magically get (yeah, promoting to a capitalist creates cash from nothing). This is required so that if a state doesn't start with capitalists and is LF it is possible for those capitalists to build a factory even though there are no current factories to provide them with an income. Usually it isn't enough and you see multiple waves of capitalist demotion/promotion cycles before they get enough money to finish the project.

Reply:
I tested it but couldn't find any magic money.
New capitalists start with the savings & cash from their previous pop. start_share has no impact on that.

Reply:
Well, I trust that the difference will start to show up when the economy is implemented and you can start doing side-by-side comparisons of Victoria 2 saves and open vic saves. Just keep it in mind for then.